### DISCUSSION TITLE
[QUESTION] Why is my snapshot/actual size this huge?

### DISCUSSION THREAD

**Discussion by leon-marzahn** [longhorn/longhorn#6625] [General]
## Question

I've got a CockroachDB pod connected to a longhorn PVC and the actual filesystem size is less than 2 GB, but for some reason the size of the snapshot is 24 GB. Is there any reason for this? I tried looking at people having the same issue with prometheus, but there shouldn't be any split data like that at all.

![image](https://github.com/longhorn/longhorn/assets/15183853/34ca4938-bd5d-4a99-8e95-171f33840301)
![image](https://github.com/longhorn/longhorn/assets/15183853/b095184b-68b2-4bde-93e7-cf5cff6ec144)

## Environment

 - Longhorn version: 1.5.0
 - Kubernetes version: 1.27.4+k3s1

**Comment by ChanYiLin** [comment]
You may find some information from this document: https://longhorn.io/kb/space-consumption-guideline/

**Comment by derekbit** [comment]
Please refer to https://longhorn.io/docs/1.5.1/volumes-and-nodes/volume-size/ as well.

**Comment by leon-marzahn** [comment]
@derekbit and @ChanYiLin Thanks for the answers! Unfortunately there are no snapshots of this volume except for one that is system hidden and 23.5 gigabytes. I've tried file system trimming as well, but that has not done anything, because I presume that those blocks were never written in the first place.

The volume head is about 900 megabytes. Is there anything else that could be causing this or I could provide to help?

**Reply by leon-marzahn** [reply]
So, I just trimmed the file system again, making sure all backups are deleted and it worked. The size is now at 5 gigabytes. Do backups count towards actual size as well?

**Reply by FaraSys** [reply]
No.
You can use Recurring Job for trim fs or backup volumes.
